src/main/java/core/basesyntax/service/file/FileReaderHandler.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.service.file;
+
+public interface FileReaderHandler {
```

Comment:
class name is overwhelmed
```suggestion
public interface FileReader {
```